app/components/createContainer/createContainerController.js

```diff
@@ -11,6 +11,11 @@ function ($scope, $state, $stateParams, Config, Info, Container, Image, Volume,
 
   $scope.imageConfig = {};
 
+  $scope.network = {
```

Comment:
Please keep using the `$scope.config.HostConfig.NetworkMode` variable to check the name of the selected network.  

A test to validate the existence of the network named `bridge` is done on L88-90 (does not exists on Windows, so we use the `nat`) and this is ignored by your implementation.

You should create a field in the `$scope.formValues` object to store the value of the 'Container' field instead of `$scope.network.Container` (try to reuse the existing objects).